tests/unit/usecases/test_list_tokens.py

```diff
@@ -0,0 +1,41 @@
+from unittest.mock import MagicMock
+
+from pytest import fixture
+
+from domain.tx.token import TokenList
+from tests.fixtures.token_factory import TokenListFactory
+from usecases.list_tokens import ListTokens
+
+
+class TestListTokens:
```

Comment:
I don't feel we need tests as we are just proxying the response for the wallet service. What do you think?